files-controlplane/etc/kubernetes/manifests/kine.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: kine
+  namespace: kube-system
+  labels:
+    component: kine
+    tier: control-plane
+spec:
+  hostNetwork: true
+  priorityClassName: system-node-critical
+  containers:
+    - name: kine-postgres
+      image: docker.io/postgres:15.2-alpine3.17
+      imagePullPolicy: IfNotPresent
+      args:
+        - --listen_addresses="127.0.0.1"
+      env:
+        - name: POSTGRES_USER
+          value: kubernetes
+        - name: POSTGRES_DB
+          value: kubernetes
+        - name: POSTGRES_PASSWORD
+          value: kubernetes
+      volumeMounts:
+        - mountPath: /var/lib/postgresql/data
+          name: postgres-data
+        - mountPath: /var/run/postgresql/
+          name: socket
+      readinessProbe:
+        failureThreshold: 1
+        periodSeconds: 5
+        exec:
+          command:
+            - sh
+            - -c
+            - pg_isready --username=$POSTGRES_USER --dbname=$POSTGRES_DB --timeout=1
+    - name: kine-kine
+      image: docker.io/rancher/kine:v0.10.0-amd64
+      imagePullPolicy: IfNotPresent
+      securityContext:
+        # kine runs as nobody by default, but on this configuration it needs to read certs from /etc/kubernetes/pki,
+        # which are root:root owned.
+        runAsUser: 0
+      args:
+        - --listen-address=0.0.0.0:2379  # Kine logs will say it is listening on localhost only. Don't believe its lies.
+        - --metrics-bind-address=127.0.0.1:8080  # By default, it would listen on all interfaces.
+        - --endpoint=postgres://kubernetes:kubernetes@/kubernetes?host=/socket
+        - --server-cert-file=/pki/server.crt
+        - --server-key-file=/pki/server.key
+      ports:
+        - containerPort: 2379
+          name: etcd
+        - containerPort: 8080
+          name: metrics
+      volumeMounts:
+        - mountPath: /pki
```

Comment:
I would use the same path that you have in the host for consistency.

Reply:
Fair enough

Reply:
After rethinking it... I'm not so convinced, seems like Mucho Texto for no reason. For consistency I'd need to also replace the postgres run directory... Meh, mucho texto.

Reply:
My way of working is always something like this:
```shell
bat manifest.yaml
# copy the path
ls -l paste-the-path
```

Also error logs will have the same path as you have in the host. It is just a nice to have.

Reply:
On the other hand, is your cluster. Your call.